src/ILCompiler/src/Program.cs

```diff
@@ -176,6 +177,9 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.DefineOptionList("initassembly", ref _initAssemblies, "Assembly(ies) with a library initializer");
                 syntax.DefineOptionList("appcontextswitch", ref _appContextSwitches, "System.AppContext switches to set");
                 syntax.DefineOptionList("runtimeopt", ref _runtimeOptions, "Runtime options to set");
+                syntax.DefineOption("methodalign", ref _methodAlignment,
+                    a => a == "min" ? MethodAlignment.Minimal : a == "opt" ? MethodAlignment.Optimal : throw new CommandLineException($"methodalign {a} not recognized"),
+                    "Sets the method alignment. Values 'min' and 'opt' are supported.");
```

Comment:
Would it make sense to also support a simple number?

Reply:
Or turn it into more general: Optimize for size / optimize for speed / default. (The JIT has flags for this, but they are not doing much today.)